stdlib/mmap.pyi

```diff
@@ -27,7 +27,7 @@ if sys.platform != "win32":
 
     PAGESIZE: int
 
-class mmap(AbstractContextManager[mmap], Iterable[int], Sized):
+class mmap(AbstractContextManager[mmap], bytearray, Iterable[int], Sized):
```

Comment:
By the way, why does `bytearray` have to be in this position in the class signature? I initially wanted to place it after `Iterable[int]` and before `Sized`, but received this error:

```
~/Development/PoCs/typing
❯ mypy --custom-typeshed-dir ../../OSS/typeshed scratch.py
scratch.py:5: error: "mmap" has no attribute "__enter__"; maybe "__iter__"?
scratch.py:5: error: "mmap" has no attribute "__exit__"
Found 2 errors in 1 file (checked 1 source file)
```